exercises/practice/pop-count/test/pop_count_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule PopCountTest do
+  use ExUnit.Case
+
+  describe "egg count" do
+    test "0 eggs" do
+      assert PopCount.egg_count(0) == 0
+    end
+
+    @tag :pending
+    test "1 egg" do
+      assert PopCount.egg_count(16) == 1
+    end
+
+    @tag :pending
+    test "4 eggs" do
+      assert PopCount.egg_count(89) == 4
+    end
+
+    @tag :pending
+    test "13 eggs" do
+      assert PopCount.egg_count(2_000_000_000) == 13
+    end
```

Comment:
@jiegillet wrote:

> (Elm compile to JS, so you know you have at most 32 bits).

You cannot make that assumption in Elixir. Maybe this is actually a good case for an Elixir-specific test, to teach people that integer can be arbitrarily big?
```suggestion
    end
    
    @tag :pending
    test "100 eggs" do
      assert PopCount.egg_count(1_267_650_600_228_229_401_496_703_205_375) == 100
    end
```

Reply:
That's a lot of eggs 🍳

Reply:
This sounds reasonable to me